addons/website_slides_survey/static/src/js/public/components/slide_upload_dialog/slide_upload_category.js

```diff
@@ -28,7 +28,16 @@ patch(SlideUploadCategory.prototype, {
             ? this.state.choices.certifications.find(
                   (c) => c.value === this.state.choices.certificationId
               ).label
-            : _t("Select or create a certification");
+            : _t("Select a certification");
+    },
```

Comment:
Hello @lm4649 
I am not sure We can do the ```super._formValidate()``` directly because the ```onClickFormSubmit``` method gives you a return and does not execute the publishing process.
```js
if (!this._formValidate()) {
     return;
}
```